core/src/components/input/input.ionic.outline.scss

```diff
@@ -128,21 +142,19 @@
   margin-inline-start: 8px;
 }
 
-// Input - Disabled
-// ----------------------------------------------------------------
+// Input Shapes
+// --------------------------------------------------
 
-:host(.input-fill-outline.input-disabled) {
```

Comment:
This can be removed because it is covered by the `:host(.input-disabled)` class since I moved the `--background` variable assignment to the proper element